test/parallel/test-cli-node-options.js

```diff
@@ -72,8 +72,8 @@ function expect(opt, want) {
     assert.ifError(err);
     if (!RegExp(want).test(stdout)) {
       console.error('For %j, failed to find %j in: <\n%s\n>',
-                    opt, expect, stdout);
-      assert(false, `Expected ${expect}`);
+                    opt, want, stdout);
+      assert(false, `Expected ${want}`);
```

Comment:
While we're in here, maybe change this to `assert.fail(...)` instead of `assert(false, ...)`?